design/media_types.go

```diff
@@ -31,3 +31,66 @@ var AuthToken = MediaType("application/vnd.authtoken+json", func() {
 		Attribute("token")
 	})
 })
+
+var WorkItem = MediaType("application/vnd.workitem+json", func() {
+	TypeName("WorkItem")
+	Description("ALM Work Item")
+	Attribute("id", String, "unique id per installation")
+	Attribute("version", Integer, "Version for optimistic concurrency control")
```

Comment:
Just FYI; Gorm has handling of CreatedAt, UpdatedAt & DeletedAt built in; http://jinzhu.me/gorm/models.html#field-createdat-used-to-store-records-created-time 

As long as the Fields exists on the struct or it extends the gorm.Model (which has those fields).